app/api/helpers/third_party_auth.py

```diff
@@ -60,7 +60,7 @@ def get_client_secret(cls):
     @classmethod
     def get_redirect_uri(cls):
         url = urllib.parse.urlparse(request.url)
-        fb_redirect_uri = url.scheme + '://' + url.netloc + '/fCallback'
+        fb_redirect_uri = 'http://localhost:4200/oauth/callback?provider=facebook'
```

Comment:
what is this? Why would you hardcode localhost:4200 on the server?

Reply:
@CosmicCoder96 Just for testing right now.